reduction/line_imaging.py

```diff
@@ -105,32 +116,48 @@
 
             dirty_tclean_made_residual = False
 
+
+            # prepare for the imaging parameters
+            pars_key = "{0}_{1}_{2}_robust{3}".format(field, band, arrayname, robust)
+            impars = line_imaging_parameters[pars_key]
+            # load in the line parameter info
+            linpars = line_parameters[line_name]
```

Comment:
or... maybe the previous comment is wrong, we don't need that, because the line parameters are independent.  OK.